### DISCUSSION TITLE
Docker Version of v0.67.0-pre.1 shows v0.66.2 in Settings

### DISCUSSION THREAD

**Discussion by inf0matics** [outline/outline#4583] [Self Hosting] [answered]
I wanted to try the latest docker version of outlinewiki/outline which is 0.67.0-pre.1.

When go to https://my-secret-outline-instance.com/settings/details --> it shows:
* v0.66.2
* 2 versions behind

<img width="227" alt="Screenshot 2022-12-16 at 09 01 57" src="https://user-images.githubusercontent.com/3327667/208051679-5bcb768a-7fe4-43fc-8919-9e88ec7800e1.png">

🐞 Bug? 🧐

**Comment by tommoor** [comment, marked as answer]
Yes, a known issue – the pre-release build was tagged wrong.
